hackathon_site/event/api_urls.py

```diff
@@ -7,5 +7,10 @@
 urlpatterns = [
     path("users/user/", api_views.CurrentUserAPIView.as_view(), name="current-user"),
     path("teams/team/", views.CurrentTeamAPIView.as_view(), name="current-team"),
```

Comment:
I just noticed that this view is in `views` instead of `api_views`, whoops. Will make a ticket to fix that

Reply:
ok, sounds good